core/tokens/legacyColors.json

```diff
@@ -1,5 +1,16 @@
 {
+  "theme-10":"blue-10",
```

Comment:
Mapping here is necessary for consumers who are using the variables in their apps, but we need to replace these values in the Core Scss files so that we do not leave legacy variables in the code.

Reply:
Fixed with new commit: updating core color variables